code/game/objects/structures/extinguisher.dm

```diff
@@ -68,7 +68,7 @@
 	return
 
 /obj/structure/extinguisher_cabinet/AltClick(mob/living/user)
-	if(user.incapacitated() || !Adjacent(user))
+	if(user.incapacitated() || !Adjacent(user) || !ishuman(user))

```

Comment:
```!istype(user)``` would be nicer, yeah animals would be able to use it, but so would drones and stuff, which atm you're excluding (robots too)